tests/testapp/models.py

```diff
@@ -8,7 +8,7 @@ class Template(ShareableOrgMixin):
 
 class Config(OrgMixin):
     name = models.CharField(max_length=16)
-    template = models.ForeignKey(Template, blank=True, null=True)
+    template = models.ForeignKey(Template, blank=True, null=True, on_delete=models.CASCADE)
```

Comment:
move the `on_delete` to the previous line to increase readability on narrow windows, eg:

```python
template = models.ForeignKey(Template, blank=True, null=True,
                             on_delete=models.CASCADE)
```